sycl/test/esimd/memory_properties.cpp

```diff
@@ -1303,4 +1303,43 @@ test_slm_gather_scatter(int byte_offset32) {
                                  props_align4);
   slm = slm_gather<float, 32, 2>(ioffset_n16_view, mask_n16, pass_thru_view,
                                  props_align4);
+
+  // Test SLM scatter using this plan:
+  // 1) slm_scatter(offsets): offsets is simd or simd_view
+  // 2) slm_scatter(offsets, mask): offsets is simd or simd_view
+  // 4) slm_scatter(...): same as (1), (2) above, but with VS > 1.
```

Comment:
```suggestion
  // 1) slm_scatter(offsets, vals): offsets/vals is simd or simd_view
  // 2) slm_scatter(offsets, vals, mask): offsets/vals is simd or simd_view
  // 3) slm_scatter(...): same as (1), (2), (3) above, but with VS > 1.
```

Please also test the cases when `slm` (aka `vals` parameter) is a `simd_view`.